html-templates/progress/section-interim-reports/reports.email.tpl

```diff
@@ -1,4 +1,4 @@
-{$from = $.User->EmailRecipient}
+{capture assign=from}no-reply@{Site::getConfig('primary_hostname')}{/capture}
```

Comment:
For both reports, the from address should be the student's advisor if the report is for a single student AND the student has an advisor, otherwise default it to the current user (sending the report)